packages/EasyStandard/docs/ecs/settings.md

```diff
@@ -5,7 +5,28 @@ is_section: true
 ---eonx_docs---
 
 ### Example of configuration
+```yaml
+# ecs.yaml
```

Comment:
Just for the heads up. The YAML configuration is now deprecated and is replaced with `ecs.php` - https://github.com/symplify/symplify/tree/master/packages/easy-coding-standard#1-create-configuration-and-setup-checkers

Reply:
Yes, that's right. But Roman advised leaving it for those projects that have not yet switched to the new configuration format.

Reply:
@TomasVotruba yep, we decided to leave it as backward compatibility for projects that have not updated ECS/Rector versions yet.

Reply:
I see. I'd put a notice above this line about YAML deprecation.

Otherwise some devs might add `ecs.yaml` to the project, then find out it's deprecated, then be really frustrated :D

Reply:
@DKeeper let's change the order (put the PHP config example first) and add the following line before the YAML config:
```
Note: The YAML configuration is deprecated in the new ECS versions. Use the PHP configuration if possible.
```

The same goes for Rector.

Reply:
Done.